src/main/java/bio/terra/cli/command/resource/Mount.java

```diff
@@ -26,6 +27,16 @@ public class Mount extends BaseCommand {
       defaultValue = "false")
   private Boolean disableCache;
 
+  /**
+   * For controlled resources created by the calling user, the mount is always read-write regardless
+   * of the value of this flag.
+   *
+   * <p>For other resources, this flag will determine whether the mount is read-only or read-write,
+   * and defaults to read-only.
+   */
+  @CommandLine.Option(names = "--read-only", description = "Mount with only read permissions.")
```

Comment:
the comment you have above is great but it's not user facing, can you put the gist of it in the description?